File: PROGRAMMERS/PY/lv.1/부족한금액계산하기.py
price = 3
money = 20
count = 4

def sol(price, money, count):
    answer = 0
    tmp = 1
    while tmp < count + 1:
        answer += price * tmp
        tmp += 1
    if answer > money:
        return answer - money
    else:
        return 0


print(sol(price, money, count))

File: PROGRAMMERS/JAVA/src/Lv_1/부족한금액계산하기.java
package Lv_1;

public class 부족한금액계산하기 {
    public long solution(int price, int money, int count) {
        long answer = -1;
        long sum = 0;
        for (int i = 1; i <= count; i++) {
            sum += price * i;
        }
        if (money - sum >= 0) {
            answer = 0;
        } else {
            answer = Math.abs(money - sum);
        }
        return answer;
    }
}
